andela-stuff/andela-teams-2
# #157719884 Implement teams page

#### What does this PR do?
Implements the addition and removal of favorite teams from the application.

#### Description of tasks to be completed?
- write reducers for handling the addition of favorite teams
- write actions for handling the addition of favorite teams
- implement component logic for handing addition and removal of favorite teams
- write HOC for protected routes
- write host variable for various environments

#### How should this be manually tested?
Open browser and navigate to loachost:8081 to start up the application
- Click on the favorites icon on one of the teams created
- You should get a toast message describing the status of your action

#### Screenshots
<img width="1440" alt="screen shot 2018-09-09 at 8 41 23 am" src="https://user-images.githubusercontent.com/24798364/45262364-2ac61800-b40d-11e8-8f00-36ee71f03ab9.png">

<img width="1440" alt="screen shot 2018-09-09 at 8 47 17 am" src="https://user-images.githubusercontent.com/24798364/45262365-36194380-b40d-11e8-890f-3af1b9068ef1.png">

<img width="1440" alt="screen shot 2018-09-09 at 8 47 28 am" src="https://user-images.githubusercontent.com/24798364/45262368-3d405180-b40d-11e8-8913-9c796e0ea01c.png">
